templates/definition/charger/twc3.yaml

```diff
@@ -5,8 +5,8 @@ products:
       generic: TWC3
 requirements:
   description:
-    en: The Tesla vehicle must be associated with the TWC3 in the evcc ui.
-    de: Das Teslafahrzeug muss dem TWC3 im evcc UI zugewiesen sein.
+    en: The Tesla vehicle must be associated with the TWC3 in the evcc ui. The Wallbox cannot be controlled directly by evcc. Control is via the vehicle. Therefore, the wall box can only be used with Tesla vehicles.
```

Comment:
It can be used with any vehicle that has start/stop/current, e.g. some Volkswagens

Reply:
OK. So what should we do ? Leave description as is?

Reply:
In any case don't make it wrong ;)

Reply:
Nvm. Seems we can only do this for Tesla right now, so this PR is correct. Thank you!